### PR TITLE
[Matrix] cleanups, icon assets add and LICENSE add

### PR DESCRIPTION
**Pull request by AlwinEsch**

### CODE REVIEW

**Comment by CvH** [comment]
@AlwinEsch as you touch every copyright header, why not also change to spdx instead to the "legacy" copyright headers? It would also stay in line with xbmc/xbmc where those change was made some months ago.

**Comment by ksooo** [comment]
+1

**Comment by AlwinEsch** [comment]
I also thought about it, but then wanted to keep it as small as possible. 
Will then tackle this in a new request round soon :smile:.

**Comment by AlwinEsch** [comment]
How we handle it with:
```cpp
/*
 *  Copyright (C) 2005-2020 Team Kodi
 *  This file is part of Kodi - https://kodi.tv <---------------------- About this
 *
 *  SPDX-License-Identifier: GPL-2.0-or-later
 *  See LICENSE.md for more information.
 */
```

The <b>`This file is part of Kodi`</b> does then only match half.

Any idea of another word?

Can start on the following addons to change them.

**EDIT:** Or like before only as ` *  https://kodi.tv`?

**Comment by AlwinEsch** [comment]
Is updated and also use now the SPDX.

**Comment by CvH** [comment]
> Any idea of another word?

@AlwinEsch just that you have an idea how this could look like, not saying this is particular the best way for that repo. There is a lot different ways of doing it. Its just an proposal, use whatever you are comfortable with :)
At LE we use
```
# SPDX-License-Identifier: GPL-2.0-or-later
# Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
```
light and easy to understand, would result into
```
 *  SPDX-License-Identifier: GPL-2.0-or-later
 *  Copyright (C) 2005-2020 Team Kodi (https://kodi.tv)
```

**Comment by AlwinEsch** [comment]
Thanks @CvH thats also a good way. But I leave it now with current. There it is then a bit equal with Kodi :smile:.

File: README.md [hunk 1]
```diff
@@ -2,14 +2,16 @@
 
 This is a [Kodi](http://kodi.tv) PVR addon for connecting to a [tvheadend](https://tvheadend.org) backend.
 
+[![License: GPL v2](https://img.shields.io/badge/License-GPL%20v2-blue.svg)](LICENSE.md)
```
Comment (CvH) [inline README.md]:
```suggestion
[![License: GPL v2+](https://img.shields.io/badge/License-GPL%20v2+-blue.svg)](LICENSE.md)
```

File: pvr.hts/addon.xml.in [hunk 2]
```diff
@@ -122,5 +122,11 @@
     <description lang="zh_CN">Tvheadend 前端，支持直播电视和录像流媒体、电子节目单、定时器</description>
     <description lang="zh_TW">Tvheadend前端，支援的串流媒體包含有直播電視和錄製節目，電子節目表，定時器 等</description>
     <platform>@PLATFORM@</platform>
+    <license>GPL-2.0</license>
```
Comment (CvH) [inline pvr.hts/addon.xml.in]:
```suggestion
    <license>GPL-2.0+</license>
```

GPL2.0 != GPL2.0+

File: pvr.hts/changelog.txt [hunk 3]
```diff
@@ -1,3 +1,9 @@
+5.4.4
+- Update Debian build code
+- Update with some minor cleanups
+- Add license name and source url to addon.xml
+- Add GPL2 license file and show GPL2 on README.md
```
Comment (CvH) [inline pvr.hts/changelog.txt]:
```suggestion
- Add GPL2+ license file and show GPL2+ on README.md
```

**Review by CvH — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AlwinEsch) [reply README.md]: There and on other places is updated, applied this also to PVR addons where requests still open.

**Pull request merged**